scripts/merge-config.escript

```diff
@@ -17,7 +17,7 @@ main(_) ->
                                case filelib:is_regular(CfgFile) of
                                    true ->
                                        {ok, Bin1} = file:read_file(CfgFile),
-                                       [Acc, io_lib:nl(), Bin1];
+                                       [Acc, "\n", Bin1];
```

Comment:
why this change? was the old could work on windows as well?

Reply:
I thought `io_lib:nl()` returns `"\r\n"` for windows.
Now that I had a look at the source, it's actually `"\n"`